### DISCUSSION TITLE
简悦 · 同步助手添加到开机自启动（Windows 端）

### DISCUSSION THREAD

**Discussion by Kenshin** [Kenshin/simpread#2011] [同步助手]
# 注意

同步助手 1.0.2 版应该解决了 Windows 无法自启动的问题，详细描述 [请看这里](https://github.com/Kenshin/simpread/discussions/2011#discussioncomment-2910717)。

以下方式仍然有效。

## 问题描述

`.exe` 其实是个自解压的程序，所以会有此问题。

## 解决方案

在文件管理器的路径栏或 Win-R 运行窗口中，输入 `%APPDATA%\Microsoft\Windows\Start Menu\Programs\Startup` 并转到这个文件夹，将同步助手程序按住右键拖过去，在这个文件夹中创建快捷方式即可。

由社区 @ttimasdf 提供解决方案。🙏

## 图文介绍

请看这里 https://github.com/Kenshin/simpread/discussions/1818

**Comment by jokervTv** [comment]
我自己摸索了另一个办法，就是把解压后的程序文件夹复制到其他非临时路径，然后打开这个新路径的程序，开机自启也正常了

**Reply by Kenshin** [reply]
嗨，谢谢提示  👍 

今天是圣诞 🎄 🎅 应该多休息。 😁  💪

**Reply by youngerruan** [reply]
为什么不用安装的形式，而是用自解压的形式呢？
按照 @jokervTv 的方法，我弄成功了，同步助手内的开机自启选项也可以用了。

**Reply by Kenshin** [reply]
@youngerruan 

> 为什么不用安装的形式，而是用自解压的形式呢？

应该是 windows 版打包的问题，它会自解压到 Windows 系统的临时目录，因为无法使用此功能。

下个版本考虑看下为什么会自解压到临时目录的问题。

**Comment by Kenshin** [comment]
> 来自 @2218675712 的图文教程

找到windows的启动文件夹

![image](https://user-images.githubusercontent.com/81074/120885048-55bf1400-c619-11eb-853e-8cd2632c5758.png)


放进去简悦的同步助手的快捷方式图标

![image](https://user-images.githubusercontent.com/81074/120885054-5bb4f500-c619-11eb-8941-7a044b20a2e7.png)

将启动后显示主界面这个选项关闭

![image](https://user-images.githubusercontent.com/81074/120885059-6079a900-c619-11eb-8249-3f69ea8112be.png)

***

下面是带有图片的说明 [详细说明](https://www.notion.so/Page-1-46d3fc28581943eb910aa677e4a7f335)

**Comment by Kenshin** [comment]
> 嗨，各位，新版同步助手来啦~

![welcome-synchelper](https://user-images.githubusercontent.com/81074/172756557-608b3e22-085f-43da-b2c7-d008503db009.png)

# 自启动

修复了 Windows 、Mac 版无法自启动的问题。

Linux 环境因为比较特殊，仅能 [使用此方式](https://github.com/Kenshin/simpread/discussions/3957) 实现。

## 下载地址

https://simpread.pro/download

## 更新说明

1. [Telegram Channel](https://t.me/simpread/407)

2. [Notion](https://simpreadpublish.notion.site/simpreadpublish/2482effc334e4df8aa3aac7da8bb8059#15e6f9d3f41f483b8e005ddf3b23e255)

## 如何升级

1. 打开同步助手后会自动提示升级。
2. 通过上面的下载地址手动升级。

# 不止如此

1.0.2 版还增加了以下新功能：

- [PDF 高级打印方式](https://github.com/Kenshin/simpread/discussions/3887)，自带目录结构、提供了定制化服务，包含主题、字体、页边距等细节。

- [日志输出](http://ksria.com/simpread/docs/#/Sync?id=%E6%97%A5%E5%BF%97)。

- 将导出文件导出到任意文件夹的 [增强导出方案](http://ksria.com/simpread/docs/#/Sync?id=%E5%A2%9E%E5%BC%BA%E5%AF%BC%E5%87%BA)。

不止如此，**优化和修复 60+ Isues**，包括：修复自启动无效、隐藏 Dock 栏，Menubar icon 颜色自适应等，墙裂建议用户升级。

# 附录

以下是自定义导出 PDF 的一些截图

![image](https://user-images.githubusercontent.com/81074/172757646-37b4e8ee-19c7-4f99-ba39-3fbdecc50c3e.png)

![image](https://user-images.githubusercontent.com/81074/172757663-f773c9aa-9650-46ec-84e2-c08e868c6244.png)

![image](https://user-images.githubusercontent.com/81074/172757678-7e00bdbe-aefc-4713-9648-55e6ff87993b.png)

# 彩蛋

同步助手 1.0.2 发布后即将进入「快车道」 更新的新版已经在测试了。😄

***

我关闭这个 Issues 了，有任何问题，仍可随时在这里讨论。😄
